diff --git a/targets.toml b/targets.toml
index 7d3432d..9147368 100644
--- a/targets.toml
+++ b/targets.toml
@@ -16,6 +16,7 @@ cpp = true
 dylib = true
 std = true
 run = true
+platforms = ["linux/arm64"]
 runners = "native qemu-user qemu-system"
 deploy = true
 
@@ -52,6 +53,7 @@ cpp = true
 dylib = true
 std = true
 run = true
+platforms = ["linux/arm64"]
 runners = "qemu-user qemu-system"
 
 [[target]]
@@ -71,6 +73,7 @@ cpp = true
 dylib = true
 std = true
 run = true
+platforms = ["linux/arm64"]
 
 [[target]]
 target = "arm-unknown-linux-gnueabihf"
@@ -79,6 +82,7 @@ cpp = true
 dylib = true
 std = true
 run = true
+platforms = ["linux/arm64"]
 
 [[target]]
 target = "armv7-unknown-linux-gnueabi"
@@ -87,6 +91,7 @@ cpp = true
 dylib = true
 std = true
 run = true
+platforms = ["linux/arm64"]
 runners = "qemu-user"
 
 [[target]]
@@ -96,6 +101,7 @@ cpp = true
 dylib = true
 std = true
 run = true
+platforms = ["linux/arm64"]
 runners = "qemu-user qemu-system"
 
 [[target]]
