diff --git a/targets.toml b/targets.toml
--- a/targets.toml
+++ b/targets.toml
@@ -28,6 +28,7 @@
 run = true
 runners = "native qemu-user"
 deploy = true
+platforms = ["linux/arm64"]
 
 [[target]]
 target = "x86_64-pc-windows-msvc"
@@ -53,6 +54,7 @@
 std = true
 run = true
 runners = "qemu-user qemu-system"
+platforms = ["linux/arm64"]
 
 [[target]]
 target = "aarch64_be-unknown-linux-gnu"
@@ -79,6 +81,7 @@
 dylib = true
 std = true
 run = true
+platforms = ["linux/arm64"]
 
 [[target]]
 target = "armv7-unknown-linux-gnueabi"
@@ -97,6 +100,7 @@
 std = true
 run = true
 runners = "qemu-user qemu-system"
+platforms = ["linux/arm64"]
 
 [[target]]
 target = "thumbv7neon-unknown-linux-gnueabihf"
