diff --git a/targets.toml b/targets.toml
index 9029850..7251822 100644
--- a/targets.toml
+++ b/targets.toml
@@ -26,6 +26,7 @@ cpp = true
 dylib = true
 std = true
 run = true
+platforms = ["linux/arm64"]
 runners = "native qemu-user"
 deploy = true
 
@@ -52,6 +53,7 @@ cpp = true
 dylib = true
 std = true
 run = true
+platforms = ["linux/arm64"]
 runners = "qemu-user qemu-system"
 
 [[target]]
@@ -61,6 +63,7 @@ cpp = true
 dylib = true
 std = true
 run = true
+platforms = ["linux/arm64"]
 runners = "qemu-user"
 build-std = true
 
@@ -71,6 +74,7 @@ cpp = true
 dylib = true
 std = true
 run = true
+platforms = ["linux/arm64"]
 
 [[target]]
 target = "arm-unknown-linux-gnueabihf"
@@ -79,6 +83,7 @@ cpp = true
 dylib = true
 std = true
 run = true
+platforms = ["linux/arm64"]
 
 [[target]]
 target = "armv7-unknown-linux-gnueabi"
@@ -87,6 +92,7 @@ cpp = true
 dylib = true
 std = true
 run = true
+platforms = ["linux/arm64"]
 runners = "qemu-user"
 
 [[target]]
@@ -96,6 +102,7 @@ cpp = true
 dylib = true
 std = true
 run = true
+platforms = ["linux/arm64"]
 runners = "qemu-user qemu-system"
 
 [[target]]
